agent/api/container/container.go

```diff
@@ -758,3 +758,19 @@ func (c *Container) ShouldCreateWithSSMSecret() bool {
 	}
 	return false
 }
+
+// MergeEnvironmentVariables appends additional envVarName:envVarValue pairs to
+// the the container's enviornment values structure
+func (c *Container) MergeEnvironmentVariables(secrets map[string]string) {
```

Comment:
I would name the method argument just `envVars` or something similar so this can be reused.

Reply:
oh true, good catch.